frontend/src/bundles/common/components/protected-route/protected-route.tsx

```diff
@@ -21,7 +21,7 @@ const ProtectedRoute: React.FC<Properties> = ({ children }) => {
         );
     }
 
-    if (!user) {
+    if (dataStatus === DataStatus.REJECTED) {
```

Comment:
```suggestion
    if (!user) {
```

Don't you feel like you're changing this line of code in a circle?